modules/spacecraftinstruments/rendering/renderablemodelprojection.cpp

```diff
@@ -126,9 +129,56 @@ RenderableModelProjection::RenderableModelProjection(const ghoul::Dictionary& di
         dictionary,
         "RenderableModelProjection"
     );
-    using ghoul::Dictionary;
-    Dictionary geometryDictionary = dictionary.value<Dictionary>(keyGeometry);
-    _geometry = modelgeometry::ModelGeometry::createFromDictionary(geometryDictionary);
+
+    if (dictionary.hasKey(KeyGeomModelFile)) {
+        std::string file;
+
+        if (dictionary.hasValue<std::string>(KeyGeomModelFile)) {
+            // Handle single file
+            file = absPath(dictionary.value<std::string>(KeyGeomModelFile));
+            _geometry = ghoul::io::ModelReader::ref().loadModel(file, false, true);
```

Comment:
For these kinds of case we have the `ghoul::Boolean` type where you can give the parameters proper names and use `Yes` and `No` instead.  If I'm just reading this, I have to go to the function declaration to see what the boolean parameters actually mean